### PR TITLE
[IMP] README.rst - Add a description for Vundle installation.

### PR DESCRIPTION
**Pull request by mmequignon**
Hi !
As @zwidny noticed in #748, there is no description in the README for Vundle install type.
This pr just adds the missing description.
Would not it be better to add a dedicated subsection for Vundle install type ?

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -91,6 +91,14 @@ Example installation command using Pathogen:
 
     git clone --recursive https://github.com/davidhalter/jedi-vim.git ~/.vim/bundle/jedi-vim
 
+Example installation using Vundle:
+
+Add the following line in you `~/.vimrc`
```
Comment (davidhalter) [inline README.rst]: Typo. :) `in your`.

I think `to your` would be better as well, but then again I'm not an English native.

Comment (mmequignon) [reply README.rst]: Yes, you're right, it's a typo.
It's fixed.

Comment (mmequignon) [reply README.rst]: Concerning the documentation file, it seems already alright.

**Comment by mmequignon** [comment]
I can squash commits before the merge if you are fine with the modifications.

**Comment by davidhalter** [comment]
I can also squash here in the web interface. No problem.

**Comment by mmequignon** [comment]
Well, that's all right for me.
That you for your quick answers !

**Comment by mmequignon** [comment]
> Do you think that it's necessary? @mmequignon
No I don't, it was just an idea.

**Comment by davidhalter** [comment]
I just realized that vundle is already documented here. https://github.com/davidhalter/jedi-vim/blob/master/doc/jedi-vim.txt

I guess in that case it makes sense to just add it to the README.

**Comment by mmequignon** [comment]
Thank you for merging !

**Pull request merged**